tools/mobavailconverter.py

```diff
@@ -81,8 +92,10 @@
 			s += '\t\tHairColorId: {}\n'.format(hair_color)
 		if cloth_color != 0:
 			s += '\t\tBodyColorId: {}\n'.format(cloth_color)
-		if gender != 0:
-			s += '\t\tGender: SEX_MALE\n'
+		if gender == 0:
+			s += '\t\tGender: "SEX_MALE"\n'
```

Comment:
shouldn't `gender == 1` be `SEX_MALE`? since previous code was != 0

Reply:
good eyes!